docs/plugins/testing_workshop_docs/Pythons-assert-keyword.md

```diff
@@ -0,0 +1,91 @@
+# Python's assert keyword
+
+## Article information
+
+This article will be more valuable if you are familiar with the Python programming language and napari software. It is the first in a series of articles on testing taken from the [January 2022 testing workshop video](https://drive.google.com/file/d/1DaMrRz-rLRQ6-_y0J8O3GRpVPCn0rgYs/view). The information in this article starts at minute 5:23. This article is a summary of the information in the video and should stand on its own. The other articles are:
+
+Article 1: This article  
+Article 2: [Pytest testing framework](./Pytest-testing-frameworks)  
+Article 3: [Readers and fixtures](./Readers-and-fixtures)  
+Article 4: [Test coverage](./Test-Coverage)  
+Article 5: Testing widgets  
+  
+### This article covers
+[Test for Pass](#test-for-pass)  
+[Test for Fail](#test-for-fail)  
+
+### Resources
+The example plugin and all the tests discussed in this article are available in [this GitHub repository](https://github.com/DragaDoncila/plugin-tests).
+  
+The key to testing in Python is the [assert](https://docs.google.com/presentation/d/1RFja0o6cZ8lAalAve8heuJ-Lrb4nOSUnfdpOSEhqqNo/edit#slide=id.g107356847e4_0_22) keyword. We *assert* a Boolean expression is true and create an error message that appears when that expression is false. 
```

Comment:
Even though I think my slides are great 😆 , I think we're better off linking out to a more comprehensive resource from here e.g. https://realpython.com/python-assert-statement/

Reply:
Done.  I put a reference to your slides in the Testing resources document.